NitroxPatcher/Patches/Dynamic/Player_WarpForward_Patch.cs

```diff
@@ -0,0 +1,26 @@
+using System.Reflection;
+using NitroxModel.Helper;
+using UnityEngine;
+
+namespace NitroxPatcher.Patches.Dynamic;
+
+public sealed partial class Player_WarpForward_Patch : NitroxPatch, IDynamicPatch
+{
+    public static readonly MethodInfo TARGET_METHOD = Reflect.Method((Player t) => t.OnConsoleCommand_warpforward(default));
+
+    public static bool Prefix(Player __instance, NotificationCenter.Notification n)
+    {
+        float num;
+        DevConsole.ParseFloat(n, 0, out num, 3f);
+        Vector3 newPosition = __instance.transform.position + __instance.camRoot.GetAimingTransform().forward * num;
+
+        if (__instance.currentMountedVehicle == null)
```

Comment:
```suggestion
        if (!__instance.currentMountedVehicle)
```
You should not use null checks against unity engine objects (see [here](https://docs.unity3d.com/6000.0/Documentation/Manual/null-reference-exception.html))